src/main/java/com/jcabi/ssh/SSH.java

```diff
@@ -243,7 +207,7 @@ public static String escape(final String arg) {
         randomize = true,
         types = IOException.class
     )
-    private Session session() throws IOException {
+    Session session() throws IOException {
```

Comment:
@kpbird retryonfailure annotation is already declared on the parent class for this method. Also, override annotation is missing